src/Core/Layer/Layer.js

```diff
@@ -71,6 +71,8 @@ function GeometryLayer(id, object3d) {
         value: id,
         writable: false,
     });
+
+    this.postUpdate = () => {};
```

Comment:
we don't have a `this.preUpdate = () => {};`. Let's either add it or have none of the 2 and guard the method call in `MainLoop` (I midly prefer the second possibility, more resilient to user-provided layers).